tools/jenkins-slave-creation-unix/conf-ubuntu-cpu-c5/install.sh

```diff
@@ -51,8 +51,8 @@ sudo -H -S -u jenkins_slave git config --global user.email "mxnet-ci"
 sudo -H -S -u jenkins_slave git config --global user.name "mxnet-ci"
 
 #Install python3, pip3 and dependencies for auto-connect.py
-sudo apt-get -y install python3 python3-pip python3-boto3 python3-jenkins python3-joblib
-sudo pip3 install "docker<4.0.0"
+sudo apt-get -y install python3 python3-pip python3-yaml python3-jenkins python3-joblib
+sudo pip3 install "docker<4.0.0" boto3
```

Comment:
I looked up diff between `apt-get install python3- ` vs `pip3 install ` here : https://askubuntu.com/questions/431780/apt-get-install-vs-pip-install
but couldn't quite get why we need to do this for boto3
Shouldn't both work?

Reply:
As an after thought `conf-ubuntu-gpu/install.sh` uses `pip3 install `. But still curious

Reply:
boto3 in ubuntu 18.04 repo is too old and doesn't support secretsmanager.

Reply:
This bug was introduced in my previous PR